src/Symfony/Component/Security/Http/EventListener/IsGrantedAttributeListener.php

```diff
@@ -47,7 +49,13 @@ public function onKernelControllerArguments(ControllerArgumentsEvent $event)
             $subject = null;
 
             if ($subjectRef) {
-                if (\is_array($subjectRef)) {
+                if ($subjectRef instanceof Expression) {
+                    $this->expressionLanguage ??= new ExpressionLanguage();
```

Comment:
This needs to give a good error message when the ExpressionLanguage component is not installed.

Reply:
Done